src/main/java/com/minecolonies/compatibility/tinkers/SlimeTreeProxy.java

```diff
@@ -6,8 +6,7 @@
 /**
  * This is the fallback for when tinkers is not present!
  */
-public class SlimeTreeProxy
-{
+public class SlimeTreeProxy {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)